.github/workflows/build-2.x.yaml

```
@@ -28,14 +28,14 @@ jobs:
           - 8
     runs-on: ${{ matrix.platform }}
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v4.1.7
```

```suggestion
      - uses: actions/checkout@v4
```
Using `@v4` will automatically use the latest minor and patch version within the v4 major version series.

---

@Bcoderx6 this comment applies to all other places in this file.